<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>GroupBox_text_box</class>
    <widget class="QGroupBox" name="GroupBox_text_box">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>300</width>
                <height>300</height>
            </rect>
        </property>
        <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
            </sizepolicy>
        </property>
        <property name="windowTitle">
            <string>GroupBox</string>
        </property>
        <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Set text box parameters&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="title">
            <string>Text Box</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
            <item>
                <widget class="QGroupBox" name="groupBox_location">
                    <property name="toolTip">
                        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Location for text label for each
                            resistivity subplot.&lt;/p&gt;&lt;p&gt;Location is in relative coordinates of the datas.&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;
                        </string>
                    </property>
                    <property name="title">
                        <string>Location</string>
                    </property>
                    <property name="flat">
                        <bool>true</bool>
                    </property>
                    <property name="checkable">
                        <bool>true</bool>
                    </property>
                    <property name="checked">
                        <bool>false</bool>
                    </property>
                    <layout class="QGridLayout" name="gridLayout">
                        <item row="1" column="0">
                            <widget class="QLabel" name="label_3">
                                <property name="toolTip">
                                    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;vertical position of the
                                        bottom of the color bar in figure between [0, 1], 0 is bottom side.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;
                                    </string>
                                </property>
                                <property name="text">
                                    <string>Y</string>
                                </property>
                            </widget>
                        </item>
                        <item row="0" column="1">
                            <widget class="QSlider" name="horizontalSlider_x">
                                <property name="toolTip">
                                    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;lateral position of the left
                                        hand corner of the color bar in figure between [0, 1], 0 is the left side.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;
                                    </string>
                                </property>
                                <property name="maximum">
                                    <number>100</number>
                                </property>
                                <property name="value">
                                    <number>90</number>
                                </property>
                                <property name="orientation">
                                    <enum>Qt::Horizontal</enum>
                                </property>
                                <property name="tickPosition">
                                    <enum>QSlider::TicksBelow</enum>
                                </property>
                                <property name="tickInterval">
                                    <number>10</number>
                                </property>
                            </widget>
                        </item>
                        <item row="0" column="2">
                            <widget class="QDoubleSpinBox" name="doubleSpinBox_x">
                                <property name="toolTip">
                                    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;lateral position of the left
                                        hand corner of the color bar in figure between [0, 1], 0 is the left side.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;
                                    </string>
                                </property>
                                <property name="maximum">
                                    <double>1.000000000000000</double>
                                </property>
                                <property name="singleStep">
                                    <double>0.100000000000000</double>
                                </property>
                                <property name="value">
                                    <double>0.900000000000000</double>
                                </property>
                            </widget>
                        </item>
                        <item row="1" column="1">
                            <widget class="QSlider" name="horizontalSlider_y">
                                <property name="toolTip">
                                    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;vertical position of the
                                        bottom of the color bar in figure between [0, 1], 0 is bottom side.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;
                                    </string>
                                </property>
                                <property name="maximum">
                                    <number>100</number>
                                </property>
                                <property name="value">
                                    <number>5</number>
                                </property>
                                <property name="orientation">
                                    <enum>Qt::Horizontal</enum>
                                </property>
                                <property name="tickPosition">
                                    <enum>QSlider::TicksBelow</enum>
                                </property>
                                <property name="tickInterval">
                                    <number>10</number>
                                </property>
                            </widget>
                        </item>
                        <item row="1" column="2">
                            <widget class="QDoubleSpinBox" name="doubleSpinBox_y">
                                <property name="toolTip">
                                    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;vertical position of the
                                        bottom of the color bar in figure between [0, 1], 0 is bottom side.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;
                                    </string>
                                </property>
                                <property name="maximum">
                                    <double>1.000000000000000</double>
                                </property>
                                <property name="singleStep">
                                    <double>0.100000000000000</double>
                                </property>
                                <property name="value">
                                    <double>0.050000000000000</double>
                                </property>
                            </widget>
                        </item>
                        <item row="0" column="0">
                            <widget class="QLabel" name="label_2">
                                <property name="toolTip">
                                    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;lateral position of the left
                                        hand corner of the color bar in figure between [0, 1], 0 is the left side.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;
                                    </string>
                                </property>
                                <property name="text">
                                    <string>X</string>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </widget>
            </item>
            <item>
                <widget class="QGroupBox" name="groupBox_2">
                    <property name="toolTip">
                        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                    </property>
                    <property name="title">
                        <string>Font</string>
                    </property>
                    <property name="flat">
                        <bool>true</bool>
                    </property>
                    <layout class="QGridLayout" name="gridLayout_2">
                        <item row="0" column="0">
                            <widget class="QCheckBox" name="checkBox_size">
                                <property name="sizePolicy">
                                    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                                        <horstretch>0</horstretch>
                                        <verstretch>0</verstretch>
                                    </sizepolicy>
                                </property>
                                <property name="toolTip">
                                    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Size of text for subplot
                                        labels&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;
                                    </string>
                                </property>
                                <property name="text">
                                    <string>Size</string>
                                </property>
                            </widget>
                        </item>
                        <item row="0" column="1">
                            <widget class="QComboBox" name="comboBox_size">
                                <property name="enabled">
                                    <bool>false</bool>
                                </property>
                                <property name="toolTip">
                                    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Size of text for subplot
                                        labels&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;
                                    </string>
                                </property>
                                <property name="currentIndex">
                                    <number>2</number>
                                </property>
                                <item>
                                    <property name="text">
                                        <string>xx-small</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>x-small</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>small</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>medium</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>large</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>x-large</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>xx-large</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>size in points</string>
                                    </property>
                                </item>
                            </widget>
                        </item>
                        <item row="0" column="2">
                            <widget class="QSpinBox" name="spinBox_size">
                                <property name="enabled">
                                    <bool>false</bool>
                                </property>
                                <property name="toolTip">
                                    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Size of text for subplot
                                        labels&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;
                                    </string>
                                </property>
                                <property name="suffix">
                                    <string>points</string>
                                </property>
                                <property name="minimum">
                                    <number>4</number>
                                </property>
                                <property name="value">
                                    <number>10</number>
                                </property>
                            </widget>
                        </item>
                        <item row="1" column="0">
                            <widget class="QCheckBox" name="checkBox_weight">
                                <property name="sizePolicy">
                                    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                                        <horstretch>0</horstretch>
                                        <verstretch>0</verstretch>
                                    </sizepolicy>
                                </property>
                                <property name="toolTip">
                                    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;weight of text label font&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                                </property>
                                <property name="text">
                                    <string>Weight</string>
                                </property>
                            </widget>
                        </item>
                        <item row="1" column="1" colspan="2">
                            <widget class="QComboBox" name="comboBox_weight">
                                <property name="enabled">
                                    <bool>false</bool>
                                </property>
                                <property name="toolTip">
                                    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;weight of text label font&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                                </property>
                                <property name="currentIndex">
                                    <number>1</number>
                                </property>
                                <item>
                                    <property name="text">
                                        <string>light</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>normal</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>medium</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>semibold</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>bold</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>heavy</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>black</string>
                                    </property>
                                </item>
                            </widget>
                        </item>
                    </layout>
                </widget>
            </item>
            <item>
                <widget class="QGroupBox" name="groupBox_padding">
                    <property name="title">
                        <string>Paddings</string>
                    </property>
                    <property name="flat">
                        <bool>true</bool>
                    </property>
                    <property name="checkable">
                        <bool>true</bool>
                    </property>
                    <property name="checked">
                        <bool>false</bool>
                    </property>
                    <layout class="QGridLayout" name="gridLayout_3">
                        <item row="0" column="0">
                            <widget class="QLabel" name="label_5">
                                <property name="toolTip">
                                    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;lateral position of the left
                                        hand corner of the color bar in figure between [0, 1], 0 is the left side.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;
                                    </string>
                                </property>
                                <property name="text">
                                    <string>X</string>
                                </property>
                            </widget>
                        </item>
                        <item row="0" column="1">
                            <widget class="QSlider" name="horizontalSlider_x_pad">
                                <property name="toolTip">
                                    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;lateral position of the left
                                        hand corner of the color bar in figure between [0, 1], 0 is the left side.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;
                                    </string>
                                </property>
                                <property name="maximum">
                                    <number>100</number>
                                </property>
                                <property name="value">
                                    <number>95</number>
                                </property>
                                <property name="orientation">
                                    <enum>Qt::Horizontal</enum>
                                </property>
                                <property name="tickPosition">
                                    <enum>QSlider::TicksBelow</enum>
                                </property>
                                <property name="tickInterval">
                                    <number>10</number>
                                </property>
                            </widget>
                        </item>
                        <item row="0" column="2">
                            <widget class="QDoubleSpinBox" name="doubleSpinBox_x_pad">
                                <property name="toolTip">
                                    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;lateral position of the left
                                        hand corner of the color bar in figure between [0, 1], 0 is the left side.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;
                                    </string>
                                </property>
                                <property name="maximum">
                                    <double>1.000000000000000</double>
                                </property>
                                <property name="singleStep">
                                    <double>0.100000000000000</double>
                                </property>
                                <property name="value">
                                    <double>0.950000000000000</double>
                                </property>
                            </widget>
                        </item>
                        <item row="1" column="0">
                            <widget class="QLabel" name="label_4">
                                <property name="toolTip">
                                    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;vertical position of the
                                        bottom of the color bar in figure between [0, 1], 0 is bottom side.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;
                                    </string>
                                </property>
                                <property name="text">
                                    <string>Y</string>
                                </property>
                            </widget>
                        </item>
                        <item row="1" column="1">
                            <widget class="QSlider" name="horizontalSlider_y_pad">
                                <property name="toolTip">
                                    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;vertical position of the
                                        bottom of the color bar in figure between [0, 1], 0 is bottom side.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;
                                    </string>
                                </property>
                                <property name="maximum">
                                    <number>100</number>
                                </property>
                                <property name="value">
                                    <number>95</number>
                                </property>
                                <property name="orientation">
                                    <enum>Qt::Horizontal</enum>
                                </property>
                                <property name="tickPosition">
                                    <enum>QSlider::TicksBelow</enum>
                                </property>
                                <property name="tickInterval">
                                    <number>10</number>
                                </property>
                            </widget>
                        </item>
                        <item row="1" column="2">
                            <widget class="QDoubleSpinBox" name="doubleSpinBox_y_pad">
                                <property name="toolTip">
                                    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;vertical position of the
                                        bottom of the color bar in figure between [0, 1], 0 is bottom side.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;
                                    </string>
                                </property>
                                <property name="maximum">
                                    <double>1.000000000000000</double>
                                </property>
                                <property name="singleStep">
                                    <double>0.100000000000000</double>
                                </property>
                                <property name="value">
                                    <double>0.950000000000000</double>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </widget>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
